### ISSUE TITLE
Steam Controller xinput doesn't work with 1.05 if this patch is installed

### ISSUE THREAD

**Issue by vgf89** [yamashi/PerformanceOverhaulCyberpunk#165]
It seems this mod breaks support for Steam Input's Xinput layer, so I can't use my Steam Controller with normal controller inputs if the patch is installed. I had to remove version.dll to prevent the patch from loading after updating from 1.04 to 1.05, and now the Steam Controller works fine.

Oddly enough, plugging in a normal Xbox One controller (without going through Steam Input) works fine with and without the mod.

**Comment by Simzy** [comment]
> Steam Input | Enables the use of more gamepads (such as the Steam gamepad)

Make sure that is set to true in the config?

**Comment by vgf89** [comment]
Here's my bin\x64\plugins\cyber_engine_tweaks\config.json (exactly the same as what's in the current release from an hour ago)

`{
    "avx": true,
    "console": true,
    "cpu_memory_pool_fraction": 0.5,
    "disable_antialiasing": false,
    "disable_async_compute": false,
    "dump_game_options": false,
    "gpu_memory_pool_fraction": 1.0,
    "memory_pool": true,
    "remove_pedestrians": false,
    "skip_start_menu": true,
    "smt": true,
    "spectre": true,
    "unlock_menu": false,
    "virtual_input": true
}
`

**Comment by hulovexu1314** [comment]
> Here's my bin\x64\plugins\cyber_engine_tweaks\config.json (exactly the same as what's in the current release from an hour ago)
> 
> ```
> {
>     "avx": true,
>     "console": true,
>     "cpu_memory_pool_fraction": 0.5,
>     "disable_antialiasing": false,
>     "disable_async_compute": false,
>     "dump_game_options": false,
>     "gpu_memory_pool_fraction": 1.0,
>     "memory_pool": true,
>     "remove_pedestrians": false,
>     "skip_start_menu": true,
>     "smt": true,
>     "spectre": true,
>     "unlock_menu": false,
>     "virtual_input": true
> }
> ```
> 
> virtual_input is true if that's what you mean.

Please check the log file or wait for repair

**Comment by vgf89** [comment]
Here's my log file. Again, on 1.05, running this mod breaks steam controller funcationality, but it works without the mod.

```
[2020-12-20 20:34:34.305] [info] Computed .text crc: 20F87F01
[2020-12-20 20:34:34.305] [info] 	Resolved to version: 1.05
[2020-12-20 20:34:34.330] [warning] 	AMD SMT Patch: failed
[2020-12-20 20:34:34.330] [warning] 	Spectre patch: failed, unknown version
[2020-12-20 20:34:34.330] [info] 	AVX Patch: skip, cpu has AVX support
[2020-12-20 20:34:34.384] [info] 	Pool patch: failed
[2020-12-20 20:34:34.397] [warning] 	Virtual Input Patch: failed
[2020-12-20 20:34:34.442] [info] 	Start screen patch: success
[2020-12-20 20:34:34.475] [info] 	Hook mouse clip function!
[2020-12-20 20:34:34.502] [info] 	Log function hook complete!
[2020-12-20 20:34:34.832] [info] 	D3D12 hook complete
[2020-12-20 20:36:00.410] [info] Computed .text crc: 20F87F01
[2020-12-20 20:36:00.410] [info] 	Resolved to version: 1.05
[2020-12-20 20:36:00.434] [warning] 	AMD SMT Patch: failed
[2020-12-20 20:36:00.434] [warning] 	Spectre patch: failed, unknown version
[2020-12-20 20:36:00.434] [info] 	AVX Patch: skip, cpu has AVX support
[2020-12-20 20:36:00.487] [info] 	Pool patch: failed
[2020-12-20 20:36:00.500] [warning] 	Virtual Input Patch: failed
[2020-12-20 20:36:00.543] [info] 	Start screen patch: success
[2020-12-20 20:36:00.577] [info] 	Hook mouse clip function!
[2020-12-20 20:36:00.603] [info] 	Log function hook complete!
[2020-12-20 20:36:00.789] [info] 	D3D12 hook complete
[2020-12-20 20:37:50.855] [info] Computed .text crc: 20F87F01
[2020-12-20 20:37:50.855] [info] 	Resolved to version: 1.05
[2020-12-20 20:37:50.880] [warning] 	AMD SMT Patch: failed
[2020-12-20 20:37:50.880] [warning] 	Spectre patch: failed, unknown version
[2020-12-20 20:37:50.880] [info] 	AVX Patch: skip, cpu has AVX support
[2020-12-20 20:37:50.937] [info] 	Pool patch: failed
[2020-12-20 20:37:50.950] [warning] 	Virtual Input Patch: failed
[2020-12-20 20:37:50.997] [info] 	Start screen patch: success
[2020-12-20 20:37:51.034] [info] 	Hook mouse clip function!
[2020-12-20 20:37:51.064] [info] 	Log function hook complete!
[2020-12-20 20:37:51.262] [info] 	D3D12 hook complete
[2020-12-20 20:40:29.909] [info] Computed .text crc: 20F87F01
[2020-12-20 20:40:29.909] [info] 	Resolved to version: 1.05
[2020-12-20 20:40:29.933] [warning] 	AMD SMT Patch: failed
[2020-12-20 20:40:29.933] [warning] 	Spectre patch: failed, unknown version
[2020-12-20 20:40:29.933] [info] 	AVX Patch: skip, cpu has AVX support
[2020-12-20 20:40:29.988] [info] 	Pool patch: failed
[2020-12-20 20:40:30.001] [warning] 	Virtual Input Patch: failed
[2020-12-20 20:40:30.046] [info] 	Start screen patch: success
[2020-12-20 20:40:30.080] [info] 	Hook mouse clip function!
[2020-12-20 20:40:30.107] [info] 	Log function hook complete!
[2020-12-20 20:40:30.280] [info] 	D3D12 hook complete
[2020-12-20 20:41:25.657] [info] Computed .text crc: 20F87F01
[2020-12-20 20:41:25.657] [info] 	Resolved to version: 1.05
[2020-12-20 20:41:25.683] [warning] 	AMD SMT Patch: failed
[2020-12-20 20:41:25.683] [warning] 	Spectre patch: failed, unknown version
[2020-12-20 20:41:25.683] [info] 	AVX Patch: skip, cpu has AVX support
[2020-12-20 20:41:25.738] [info] 	Pool patch: failed
[2020-12-20 20:41:25.751] [warning] 	Virtual Input Patch: failed
[2020-12-20 20:41:25.795] [info] 	Start screen patch: success
[2020-12-20 20:41:25.833] [info] 	Hook mouse clip function!
[2020-12-20 20:41:25.864] [info] 	Log function hook complete!
[2020-12-20 20:41:26.068] [info] 	D3D12 hook complete
[2020-12-20 20:48:12.638] [info] Computed .text crc: 20F87F01
[2020-12-20 20:48:12.638] [info] 	Resolved to version: 1.05
[2020-12-20 20:48:12.664] [warning] 	AMD SMT Patch: failed
[2020-12-20 20:48:12.664] [warning] 	Spectre patch: failed, unknown version
[2020-12-20 20:48:12.664] [info] 	AVX Patch: skip, cpu has AVX support
[2020-12-20 20:48:12.721] [info] 	Pool patch: failed
[2020-12-20 20:48:12.734] [warning] 	Virtual Input Patch: failed
[2020-12-20 20:48:12.780] [info] 	Start screen patch: success
[2020-12-20 20:48:12.817] [info] 	Hook mouse clip function!
[2020-12-20 20:48:12.847] [info] 	Log function hook complete!
[2020-12-20 20:48:13.030] [info] 	D3D12 hook complete
[2020-12-20 22:18:57.088] [info] Game version 1.5
[2020-12-20 22:18:57.128] [info] 	AMD SMT Patch: success
[2020-12-20 22:18:57.128] [warning] 	Spectre patch: failed, unknown version
[2020-12-20 22:18:57.143] [warning] 	Virtual Input Patch: failed
[2020-12-20 22:18:57.187] [info] 	Start screen patch: success
[2020-12-20 22:18:57.223] [info] 	Hook mouse clip function!
[2020-12-20 22:18:57.252] [info] 	Log function hook complete!
[2020-12-20 22:18:57.505] [info] 	D3D12 hook complete
```

**Comment by brettp** [comment]
I am seeing the exact same thing on the Steam version. @yamashi - Does closing this ticket mean the problem was already addressed?
